docs/docs/getting-started/deployment/configuration.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Configuration
+sidebar_label: Configuration
+---
+
+## Airy Core Configuration File
+
+The `infrastructure/airy.tpl.yaml` file contains examples of all possible configuration options. Copy and rename it to `infrastructure/airy.yaml` and modify it according to your environment:
+
+```bash
+cd infrastructure
+cp airy.tpl.yaml airy.yaml
+```
+
+We will guide you through the different sections so you can make the changes you are looking for. Keys can also be omited for services which you do not wish to configure.
+
+### Global
+
+- **appImageTag**
+
+  image tag of the container images for the `Airy Components`
+
+- **containerRegistry**
+
+  The images of the `Airy Components` are stored in the [Github Container Registry](https://docs.github.com/en/packages/guides/about-github-container-registry).
+
+- **namespace**
+
+  Kubernetes namespace the `Airy Core` will use
+
+### Prerequisites
+
+These settings are used to connect the `Airy Components` to your Kafka cluster, PostgreSQL and Redis.
+
+- **kafka**
+
+  `commitInterval` sets the [Kafka Commit Interval](https://kafka.apache.org/documentation/#consumerconfigs_auto.commit.interval.ms) if you are using the included Helm chart.
+
+- **redis**
+- **postgres**
+
+  We recommend that you create a new database if you are reusing a PostgreSQL server to avoid name collisions.
+
+### Components
+
+- **api**
+
+  `mail*` settings are needed if you want the Airy Core to send emails out
+
+  `jwtSecret` should be set to a long secure secret in production environments
```

Comment:
this is the HTTP api JWT signing key right? I think the name has to be more explicit (I get we'd have to change the code... but if not now when? :))